src/hooks/useAccessControl/index.test.jsx

```diff
@@ -76,7 +81,7 @@ describe("useAccessControl", () => {
     await waitForNextUpdate();
     expect(result.current.accessControl).not.toBeNull();
     rerender(resourceInfo + 2);
-    expect(result.current.accessControl).toBeNull();
+    expect(result.all[0].accessControl).toBeNull(); // checking that the first render sets accessControl to null
```

Comment:
I don't really understand why this change is necessary - why shouldn't the last render set it to null? (Ideally, the comment describes the why, rather than the what.)

Reply:
I found that `current` checks the current render, which is actually after the data is available and returned, so I had to checked what happened before the data was requested and available, since the test is to check that it is set to null while loading.